### PR TITLE
Use the full lodash package

### PR DESCRIPTION
**Pull request by yjpa7145**
Because the full lodash package is being installed as a dependency of other packages anyway, we aren't saving ourselves anything by using the `lodash.xxx` packages. This updates our code to rely on `lodash`. By importing lodash functions using `require('lodash/xxx')`, it keeps the webpack size small.

### CODE REVIEW

File: example/spec/parallel/syncGranule/SyncGranuleSuccessSpec.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const fs = require('fs');
-const difference = require('lodash.difference');
+const { difference } = require('lodash/difference');
```
Comment (nemreid) [inline example/spec/parallel/syncGranule/SyncGranuleSuccessSpec.js:2]: Shouldn't this be ...?
```suggestion
const difference = require('lodash/difference');
```

File: tasks/move-granules/index.js [hunk 2]
```diff
@@ -146,7 +145,7 @@ async function moveFileRequest(
   };
 
   // the file moved to destination
-  const fileMoved = clonedeep(file);
+  const fileMoved = { ...file };
```
Comment (nemreid) [inline tasks/move-granules/index.js:148]: Same note as the other file.

File: example/spec/parallel/syncGranule/SyncGranuleSuccessSpec.js (hunk 1 above)
Comment (nemreid) [reply example/spec/parallel/syncGranule/SyncGranuleSuccessSpec.js:2]: The export is `export default difference`

File: .eslintrc.json [hunk 3]
```diff
@@ -121,22 +121,14 @@
...
-    {
-      "files": ["**/bin/**/*.js"],
-      "rules": {
-        "no-console": "off"
-      }
+      "files": [
+        "**/bin/**/*.js",
+        "**/spec/**/*.js",
+        "packages/api/migrations/**/*.js",
+        "packages/deployment/**/*.js",
```
[the remark this answers is not in the document]
Comment (yjpa7145) [reply .eslintrc.json:128]: Yup.

File: tasks/test-processing/index.js [hunk 4]
```diff
@@ -14,11 +13,12 @@ async function uploadFakeBrowse(input) {
   input.granules.forEach((granule) => {
     granule.files.forEach((file) => {
       if (file.type === 'data') {
-        const browseFile = cloneDeep(file);
+        const browseFile = { ...file };
```
[the remark this answers is not in the document]
Comment (yjpa7145) [reply tasks/test-processing/index.js:16]: Agreed, they do behave differently. In this case, though, that isn't a problem.

File: tasks/move-granules/index.js (hunk 2 above)
Comment (yjpa7145) [reply tasks/move-granules/index.js:148]: Yeah, I checked first and mad sure that this behaves as expected.

**Pull request merged**